gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -108,16 +113,6 @@ static Sum& operator+=(Sum& sum, const GaussianFactor::shared_ptr& factor) {
 }
 
 Sum HybridFactorGraph::sum() const {
-  if (nrNonlinearFactors()) {
```

Comment:
Why remove?

Reply:
This is for elimination. The test throws this error for the Switching test fixture.